### PR TITLE
i18n(ko-KR): update `api-reference.mdx`

### PR DESCRIPTION
**Pull request by jsparkdev**
#### Description (required)

- update `api-reference.mdx`
- #9240 
- #10167

#### Related issues & labels (optional)

- Closes #
- Suggested label:

### CODE REVIEW

**Comment by jsparkdev** [comment]
Once #10204, #10153, #10177, #10188 are merged, I expect to be able to merge this PR successfully.

**Review by yanthomasdev — APPROVED** [review]
LGTM! I'll merge the umbrella PR once I approve all of the individual PRs.

**Pull request closed without merging**
